src/components/button/button.tsx

```diff
@@ -87,19 +88,21 @@ export class Button
    */
   @Prop() readonly size: "large" | "normal" | "small" = "normal";
 
-  /**
-   * Emitted when the element is clicked.
-   */
-  @Event() gxClick: EventEmitter;
-
+  @Listen("click", { capture: true })
```

Comment:
I think that with this approach, when `disabled=false`, the `gxClick` event will fire twice. Once in the capture phase and once in the bubbling phase. Is that right?

Additionally, once the click is cancelled when `disabled` is `true`, does having the `gxClick` event make sense?

Reply:
Your approach is accurate. I tested and it emits `gxClick` twice. 
Look at the next commit. I cancel the event propagation, and now, event target is emitted from gxButton.

Reply:
It looks better now! However, do we need the `gxClick` event now that the `click` event is canceled when `disabled` is `true`?

Reply:
@ncardeli removing `gxClick` event, there is a conflict with the interface of `ClickableComponent`

Reply:
Yes, exactly. Feel free to remove that interface from the component.